primitives/src/policy.rs

```diff
@@ -445,16 +501,161 @@ impl Policy {
     /// I.e 1 means that the full rewards should be given, whereas 0.5 means that half of the rewards should be given
     /// The input to this function is the batch delay, in milliseconds
     /// The function is: [(1 - MINIMUM_REWARDS_PERCENTAGE) * e ^(-BLOCKS_DELAY_DECAY * t^2)] + MINIMUM_REWARDS_PERCENTAGE
+    #[cfg_attr(feature = "ts-types", wasm_bindgen(js_name = batchDelayPenalty))]
     pub fn batch_delay_penalty(delay: u64) -> f64 {
         let t = delay as f64;
         let exponent = -Self::BLOCKS_DELAY_DECAY * t * t;
 
         (1.0 - Self::MINIMUM_REWARDS_PERCENTAGE) * exponent.exp() + Self::MINIMUM_REWARDS_PERCENTAGE
     }
+}
 
-    #[inline]
-    pub fn get_or_init(policy: Policy) -> Policy {
-        *GLOBAL_POLICY.get_or_init(|| policy)
+// wasm_bindgen does not support exposing `pub const` struct fields, so we reimplement those consts
+// as getters when compiling for WASM.
+#[cfg(feature = "ts-types")]
+#[wasm_bindgen]
+#[allow(non_snake_case)]
+impl Policy {
+    /// This is the address for the staking contract. Corresponds to
+    /// 'NQ77 0000 0000 0000 0000 0000 0000 0000 0001'
+    #[cfg_attr(feature = "ts-types", wasm_bindgen(getter = STAKING_CONTRACT_ADDRESS))]
+    pub fn WASM_STAKING_CONTRACT_ADDRESS() -> String {
```

Comment:
Can we actually have the function name in lower case (and keeping the `wasm_bindgen` `getter` property om upper case)?

Reply:
Yes

Reply:
Done in 70eacbe9323e7c8223dde6ff57b78a86cc8fe055